### ISSUE TITLE
ranef not defined for GLMM

### ISSUE THREAD

**Issue by dhb2128** [JuliaStats/MixedModels.jl#294]
Related to #187, I cannot retrieve the random intercepts when using GLMMs, although I can with LMMs. Is there something that I'm doing wrong, or is this still a bug?

```julia
datf = joinpath(dirname(pathof(MixedModels)),"..","test","dat.rda")
const dat = Dict(Symbol(k)=>v for (k,v) in load(datf))
verbaggform = @formula(r2 ~ 1+a+g+b+s+m+(1|id)+(1|item));
gm1 = fit(MixedModel, verbaggform, dat[:VerbAgg], Bernoulli())
```

```traceback
julia> gm1 = fit(MixedModel, verbaggform, dat[:VerbAgg], Bernoulli())
Generalized Linear Mixed Model fit by maximum likelihood (nAGQ = 1)
  r2 ~ 1 + a + g + b + s + m + (1 | id) + (1 | item)
  Distribution: Bernoulli{Float64}
  Link: LogitLink()

  Deviance: 8135.8329

Variance components:
          Column   Variance  Std.Dev.
 id   (Intercept)  1.793473 1.33920611
 item (Intercept)  0.117149 0.34227036
 Number of obs: 7584; levels of grouping factors: 316, 24

Fixed-effects parameters:
─────────────────────────────────────────────────────
              Estimate  Std.Error    z value  P(>|z|)
─────────────────────────────────────────────────────
(Intercept)   0.553512  0.368905     1.50042   0.1335
a             0.05742   0.0160373    3.58041   0.0003
g: M          0.320766  0.183041     1.75243   0.0797
b: scold     -1.05987   0.176294    -6.01194   <1e-8
b: shout     -2.10387   0.178552   -11.783     <1e-31
s: self      -1.05433   0.144737    -7.28446   <1e-12
m: do        -0.707051  0.144558    -4.89112   <1e-5
─────────────────────────────────────────────────────

julia> ranef(gm1)
ERROR: MethodError: no method matching ranef(::GeneralizedLinearMixedModel{Float64})
Closest candidates are:
  ranef(::LinearMixedModel{T}; uscale) where T at /home/dhb13/.julia/packages/MixedModels/4DVaj/src/linearmixedmodel.jl:437
Stacktrace:
 [1] top-level scope at REPL[6]:1

julia> ranef(gm1, named=true)
ERROR: MethodError: no method matching ranef(::GeneralizedLinearMixedModel{Float64}; named=true)
Closest candidates are:
  ranef(::LinearMixedModel{T}; uscale) where T at /home/dhb13/.julia/packages/MixedModels/4DVaj/src/linearmixedmodel.jl:437 got unsupported keyword argument "named"
Stacktrace:
 [1] top-level scope at REPL[7]:1
```

Here are my current package versions:
```julia
pkg> status MixedModels
    Status `~/.julia/environments/v1.3/Project.toml`
  [324d7699] CategoricalArrays v0.7.7
  [31c24e10] Distributions v0.21.12
  [38e38edf] GLM v1.3.7
  [ff71e718] MixedModels v2.1.2
  [2913bbd2] StatsBase v0.32.1
  [bd369af6] Tables v1.0.2
```

**Comment by palday** [comment]
This is just an oversight in providing the extractor method. For now, you can get them with:
```julia
ranef(gm1.LMM)
```

There are a few nice compatibility fixes in v2.2 and a LOT of improvements in `#master`, so consider upgrading. :)

**Comment by dhb2128** [comment]
Thanks for the clarification. I upgraded to v2.2 and calling it via `gm1.LMM` works. Feel free to close.
